h/tasks/url_migration.py

```diff
@@ -0,0 +1,14 @@
+# pylint: disable=no-member # Instance of 'Celery' has no 'request' member
+from h.celery import celery
+
+
+@celery.task
+def move_annotations_by_url(old_url, new_url_info):
+    migration_svc = celery.request.find_service(name="url_migration")
+    migration_svc.move_annotations_by_url(old_url, new_url_info)
```

Comment:
I have used the term "move annotations" for these tasks and method names in `URLMigrationService`, because "moving an annotation between URLs" is how support will likely think about the operation, though earlier drafts used "update annotation URLs" in some places.